recipes/base58/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "base58" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c83584a8b917dc52dd634307137f2ad2721a9efb4f1de32fc7eaaaf87844177e
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - base58 = base58.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - base58
+  commands:
+    - pip check
+    - base58 --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/keis/base58
+  summary: Base58 and Base58Check implementation
+  license: MIT
+  license_file: COPYING 
```

Comment:
Please include a comment above the `license_file` line with the following url for a pull request to add the license file to the source distribution. This will be a reminder to check the status of whether it has been included in a future release so the manually added copy can be removed.

https://github.com/keis/base58/pull/55

Reply:
My bad, I think it is included, it is just called copying. https://github.com/keis/base58/blob/master/COPYING (and like with eth-typing, I thought I would have to be included separately)

Reply:
The `COPYING` file is not included in the source distribution (I just checked the tarball from pypi). If you could please add the comment in the `meta.yaml` that I requested, I'd be happy to merge this

Reply:
Changed the tarball to github, which has the license file.